including sys/socket.h panics

Just including `sys/socket.h` causes a panic:
```
#include <sys/socket.h>

int main() {
  inet_addr("...");
  return 0;
}
```
I confirmed that `arpa/inet.h `is being registered in `registery.go`. I'm not sure the difference between Env.GetLibrary and Env.GetLib.
```
panic: cannot find library: arpa/inet.h

goroutine 38 [running]:
github.com/gotranspile/cxgo/libs.(*Env).GetLib(...)
        /Users/jarrettkuklis/GolandProjects/cxgo/libs/config.go:58
github.com/gotranspile/cxgo/libs.init.29.func1(0xc000140320, 0xc000114c60)
        /Users/jarrettkuklis/GolandProjects/cxgo/libs/sys_socket.go:20 +0x3211
github.com/gotranspile/cxgo/libs.(*Env).GetLibrary(0xc000140320, 0xc00002a051, 0xc, 0xc00039c601, 0xc00039e2a8)
        /Users/jarrettkuklis/GolandProjects/cxgo/libs/registry.go:140 +0xe2
github.com/gotranspile/cxgo/libs.(*Env).NewLibrary(0xc000140320, 0xc00002a040, 0x1d, 0xc0001e3938, 0x10ad98d)
        /Users/jarrettkuklis/GolandProjects/cxgo/libs/registry.go:193 +0xcf
github.com/gotranspile/cxgo.includeFS.content(0xc000140320, 0xc00002a040, 0x1d, 0xc00039c601, 0xc0001e3978, 0xc0001e3978, 0x10a6707, 0x1585b60)
        /Users/jarrettkuklis/GolandProjects/cxgo/vfs.go:28 +0x4c
github.com/gotranspile/cxgo.includeFS.Stat(0xc000140320, 0xc00002a040, 0x1d, 0xc000110001, 0x1, 0x0, 0x1585d20, 0xc00039c660)
        /Users/jarrettkuklis/GolandProjects/cxgo/vfs.go:36 +0x52
modernc.org/cc/v3.overlayFS.Stat(0x15891b0, 0x1805848, 0x1589138, 0xc000140320, 0xc00002a040, 0x1d, 0x1, 0x0, 0x1d, 0xc00002a040, ...)
        /Users/jarrettkuklis/go/pkg/mod/modernc.org/cc/v3@v3.33.10-0.20210803134125-62ecad85ad9f/filesystem.go:87 +0x13b
modernc.org/cc/v3.(*context).statFile(0xc000104600, 0xc00002a040, 0x1d, 0xc00002a001, 0x1d, 0x0, 0x1585b60, 0xc000110030)
        /Users/jarrettkuklis/go/pkg/mod/modernc.org/cc/v3@v3.33.10-0.20210803134125-62ecad85ad9f/cc.go:769 +0x5e
modernc.org/cc/v3.(*ppIncludeDirective).translationPhase4(0xc00015c740, 0xc00015a840)
        /Users/jarrettkuklis/go/pkg/mod/modernc.org/cc/v3@v3.33.10-0.20210803134125-62ecad85ad9f/cpp.go:2462 +0xa6c
modernc.org/cc/v3.(*ppFile).translationPhase4(0xc000140440, 0xc00015a840)
        /Users/jarrettkuklis/go/pkg/mod/modernc.org/cc/v3@v3.33.10-0.20210803134125-62ecad85ad9f/cpp.go:3029 +0xa3
modernc.org/cc/v3.(*cpp).translationPhase4.func2(0xc00015a840, 0xc000140480, 0x2, 0x2)
        /Users/jarrettkuklis/go/pkg/mod/modernc.org/cc/v3@v3.33.10-0.20210803134125-62ecad85ad9f/cpp.go:2598 +0x1b2
created by modernc.org/cc/v3.(*cpp).translationPhase4
        /Users/jarrettkuklis/go/pkg/mod/modernc.org/cc/v3@v3.33.10-0.20210803134125-62ecad85ad9f/cpp.go:2586 +0x126
```

--- a/libs/config.go
+++ b/libs/config.go
@@ -52,7 +52,10 @@ func (c *Env) ResolveImport(name string) string {
 	return path
 }
 
-func (c *Env) GetLib(name string) *Library {
+// LookupLibrary finds an already loaded Library. It is useful to prevent import loops.
+//
+// Typically, the GetLibrary function should be used instead, because it will load the library automatically, if needed.
+func (c *Env) LookupLibrary(name string) *Library {
 	l, ok := c.libs[name]
 	if !ok {
 		panic(errors.New("cannot find library: " + name))

--- a/libs/pthread.go
+++ b/libs/pthread.go
@@ -15,8 +15,7 @@ func init() {
 		intT := types.IntT(4)
 		argT := c.PtrT(nil)
 		retT := c.PtrT(nil)
-		timeLib := c.GetLib(timeH)
-		timespecT := timeLib.GetType("timespec")
+		timespecT := c.GetLibraryType(timeH, "timespec")
 		threadT := types.NamedTGo("pthread_t", "pthread.Thread", c.MethStructT(map[string]*types.FuncType{
 			"Join":        c.FuncTT(intT, c.PtrT(retT)),
 			"TimedJoinNP": c.FuncTT(intT, c.PtrT(retT), c.PtrT(timespecT)),

--- a/libs/registry.go
+++ b/libs/registry.go
@@ -129,6 +129,7 @@ var defPathReplacer = strings.NewReplacer(
 	".", "_",
 )
 
+// GetLibrary finds or initializes the library, given a C include filename.
 func (c *Env) GetLibrary(name string) (*Library, bool) {
 	if l, ok := c.libs[name]; ok {
 		return l, true
@@ -177,6 +178,7 @@ func (c *Env) GetLibrary(name string) (*Library, bool) {
 	return l, true
 }
 
+// GetLibraryType is a helper for GetLibrary followed by GetType.
 func (c *Env) GetLibraryType(lib, typ string) types.Type {
 	l, ok := c.GetLibrary(lib)
 	if !ok {

--- a/libs/sys_socket.go
+++ b/libs/sys_socket.go
@@ -17,7 +17,7 @@ func init() {
 		gintT := c.Go().Int()
 		uintT := types.UintT(4)
 		bytesT := c.C().String()
-		inAddrT := c.GetLib(arpaInetH).GetType("in_addr")
+		inAddrT := c.GetLibraryType(arpaInetH, "in_addr")
 		sockAddrT := types.NamedT("cnet.SockAddr", types.StructT([]*types.Field{
 			{Name: types.NewIdentGo("sa_family", "Family", int16T)},
 			{Name: types.NewIdentGo("sa_data", "Data", c.C().BytesN(14))},

--- a/libs/sys_time.go
+++ b/libs/sys_time.go
@@ -12,7 +12,7 @@ const (
 func init() {
 	RegisterLibrary(sysTimeH, func(c *Env) *Library {
 		intT := types.IntT(4)
-		timeLib := c.GetLib(timeH)
+		timeLib := c.LookupLibrary(timeH)
 		timevalT := timeLib.GetType("timeval")
 		timespecT := timeLib.GetType("timespec")
 		_ = timespecT

--- a/libs_test.go
+++ b/libs_test.go
@@ -61,6 +61,22 @@ func foo() {
 `,
 	},
 	{
+		name: "sys socket",
+		src: `
+#include <sys/socket.h>
+
+void foo() {
+	in_addr_t a = inet_addr("1.2.3.4");
+}
+`,
+		exp: `
+func foo() {
+	var a cnet.Addr = cnet.ParseAddr("1.2.3.4")
+	_ = a
+}
+`,
+	},
+	{
 		name: "math",
 		src: `
 #include <math.h>

